src/js/common/jsonify.js

```diff
@@ -0,0 +1,34 @@
+// Handlebars helper to allow us to output JSON structures as part of rendered content.
+//
+// This can be accessed in your markup using syntax like:
+//
+// `{{{jsonify VARIABLE}}}`
+//
+// The triple braces are advisable to avoid escaping ampersands.
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.templates.hb.helper.jsonify");
+
+
+gpii.templates.hb.helper.jsonify.getJsonifyFunction = function () {
+    return function (context) {
+        try {
+            return JSON.stringify(context);
```

Comment:
I have done this.  This revealed a problem with my client tests which could not handle matching multiline values.  I refactored that part of the tests to be less brittle, that check now works with the client DOM.